marklogic-data-hub/src/test/java/com/marklogic/hub/util/Installer.java

```diff
@@ -38,6 +38,11 @@ public void uninstallHub() {
         if (htb.isCertAuth() || htb.isSslRun()) {
         	htb.sslCleanup();
         }
-        htb.deleteProjectDir();
+        try {
+        	htb.deleteProjectDir();
+        }
+        catch(Exception e) {
+        	//don't do anything
+        }
```

Comment:
Shouldn't we capture what the error was if the test fails due to an IO issue?

Reply:
I will try to find out the underlying reason why that folder wasn't deleted, else, I will print the exception